Example/PubNub/PNAppDelegate.m

```diff
@@ -77,7 +77,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 - (void)pubNubInit {
 
     // Initialize PubNub client.
-    self.myConfig = [PNConfiguration configurationWithPublishKey:_pubKey subscribeKey:_subKey];
+    self.myConfig = [PNConfiguration configurationWithPublishKey:_pubKey subscribeKey:_subKey uuid:@"pubnub"];
```

Comment:
Was including "pubnub" as a default `uuid` intentional?

Reply:
@CraigLn  `uuid` is **required** field. Later on those values will be changed to random, before client instance allocation.
This example app need really big clean up :)